### PR TITLE
Setup serving static assets, place route, web server

### PR DESCRIPTION
**Pull request by shea-hawkins**

### CODE REVIEW

**Comment by ccarey221** [comment]
🚢

File: server/DOCKERFILE [hunk 1]
```diff
@@ -0,0 +1,11 @@
+FROM node:6
+MAINTAINER Hadeshco
```
Comment (hannahhenderson) [inline server/DOCKERFILE]: * Hadashco

**Comment by hannahhenderson** [comment]
LGTM as foundational piece. Let's make sure to include tests in future.

**Comment by Dane456** [comment]
🚢

**Pull request merged**